docs/sources/reference/builder.md

```diff
@@ -322,6 +322,17 @@ default specified in `CMD`.
 > the result; `CMD` does not execute anything at build time, but specifies
 > the intended command for the image.
 
+## LABEL
+   LABEL <key>=<value> <key>=<value> <key>=<value> ...
+
+ --The `LABEL` instruction allows you to describe the image your `Dockerfile`
+is building. `LABEL` is specified as name value pairs. This data can 
+be retrieved using the `docker inspect` command
+
+
+    LABEL Description="This image is used to start the foobar executable" Vendor="ACME Products"
+    LABEL Version="1.0"
+
```

Comment:
@SvenDowideit The idea is that these labels should be conceptually the same as the host labels.  Going forward we will just be able to apply labels to everything, on host, containers, images, and future networks and volumes.